Runtime/Managers/Manager.cs

```diff
@@ -40,38 +42,85 @@ public abstract class Manager : MonoBehaviour
             return(s_Managers.ContainsKey(typeof(T)));
         }
 
-        static readonly Type[] kAllManagerTypes = TypeUtility.GetConcreteTypes<Manager>();
+        static readonly Type[] kAllManagerTypes = TypeUtility.GetConcreteTypes<Manager>();   
 
         [RuntimeInitializeOnLoadMethod(RuntimeInitializeLoadType.BeforeSceneLoad)]
         static void AutoCreateAll()
         {
             s_Managers.Clear();
 
-            var exclusionList = GameplayIngredientsSettings.currentSettings.excludedeManagers;
+            // Get the current scene Resources/ path
+            string sceneAssetPath = PathUtils.GetCurrentSceneResourcesPath();
```

Comment:
This scene asset path will ever be the one from your first boot scene as `AutoCreateAll()` is only invoked at game startup, I don´t quite understand the purpose of it.